buf.gen.yaml

```diff
@@ -9,17 +9,13 @@ managed:
 plugins:
   - name: go
     opt: paths=source_relative
-    out: ./perseusapi
-  - name: go-grpc
+    out: perseusapi
+  - name: connect-go
     opt:
       - paths=source_relative
-    out: ./perseusapi
-  - name: grpc-gateway
-    opt:
-      - paths=source_relative
-      - allow_repeated_fields_in_body=true
-    out: ./perseusapi
+    out: perseusapi
   - name: openapiv2
```

Comment:
A note here: Vanguard does not currently have its own OpenAPI/Swagger generator so we're still using gRPC Gateway's `protoc-gen-openapiv2`.

Reply:
Thanks for the tip.  I'll take a look and see if that's an acceptable replacement.

Reply:
Give it a whirl and `protoc-gen-connect-openapi` outputs "pure" OpenAPI, which is not quite the same thing as Swagger.  Haven't had time to do a deep comparison to see how different things are and whether or not those differences matter for things like Swagger UI.

Gonna leave that for a follow-up PR, though.